src/Contracts/ClientContract.php

```diff
@@ -1,132 +1,84 @@
 <?php
```

Comment:
Looks like some accidental changes in this file. We should expect to only see the new ResponsesContract added. The other changes I don't expect.